pkgs/by-name/on/oniux/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitLab,
+}:
+rustPlatform.buildRustPackage rec {
```

Comment:
Consider using `finalAttrs` for improved flexibility on `override`

```suggestion
rustPlatform.buildRustPackage (finalAttrs {
```

Reply:
done